src/components/Tabs.react.js

```diff
@@ -121,6 +121,96 @@ EnhancedTab.defaultProps = {
     },
 };
 
+const NextTab = ({
```

Comment:
Consider wrapping in `React.memo` to make this component Pure.
https://reactjs.org/docs/react-api.html#reactmemo